openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -612,6 +612,43 @@ if (typeof OpenAssessment.Server === "undefined" || !OpenAssessment.Server) {
             $.ajax({
                 type: "POST", url: url, data: payload, contentType: jsonContentType
             });
-        }
-    };
+        },
+
+        /**
```

Comment:
I would imagine the comments will get filled out with details of return values, etc..